samples/semantickernel-concepts/semantickernel-syntax-examples/src/main/java/com/microsoft/semantickernel/samples/syntaxexamples/memory/AzureAISearchVectorStore.java

```diff
@@ -136,18 +136,26 @@ public static void dataStorageWithAzureAISearch(
             .then(storeData(collection, embeddingGeneration, sampleData()))
             .block();
 
-        // Query the Azure AI Search client for results
-        // This might take a few seconds to return the best result
-        var result = searchClient.getSearchAsyncClient(collectionName)
-            .search("How to get started with the Semantic Kernel?")
-            .blockFirst();
+        // Search for results
+        // Might need to wait for the data to be indexed
+        var results = search("How to get started", collection, embeddingGeneration).block();
+        var searchResult = results.get(0);
```

Comment:
Is it possible for an empty result (would raise ArrayIndexOutOfBounds)? Possible for results to be null?

Reply:
Only way to happen is if the user deletes all records from the index. I added a check anyways, thanks